models/components/versions/base/component.go

```diff
@@ -19,4 +19,9 @@ type Satisfies interface {
 	GetControlKey() string
 	GetNarrative() string
 	GetCoveredBy() common.CoveredByList
+}
+
+// Base is the common struct that all component schemas must have.
```

Comment:
Ahh actually I need to fix this comment.

In a previous iteration of this, I was embedding the `Base` in all the versions of `Component` But here and in the future PRs, I won't be anymore. I don't need to / can't anymore since I'm doing that Unmarshal trick in #158 anyhow.

Needs to read `Base is the bare minimum that every Component will have and is used to find the semver Version`.